### DISCUSSION TITLE
elevation in station_stack()

### DISCUSSION THREAD

**Discussion by Dominatorexpress** [PyGLImER/PyGLImER#137] [Q&A] [answered]
Hello everyone!

At first, I would like to present my, congratulations to the team that produced this tool!

I have a little question on the command station_stack() command which allows to obtain the amplitude according to the depth.
Indeed, this command takes into consideration the elevation of the station?
I checked the code and didn't see if that was the case. 

Thanks 

Sincerely

**Comment by lsawade** [comment, marked as answer]
Hi @Dominatorexpress,

The crucial functions are the "time table" functions:

[`dt_table(...)`&#8599;](https://github.com/PyGLImER/PyGLImER/blob/673dd754fe49b8707f0ee72673b395c10e34e1ac/src/pyglimer/rf/moveout.py#L399)

[`dt_table_3D(...)`&#8599;](https://github.com/PyGLImER/PyGLImER/blob/673dd754fe49b8707f0ee72673b395c10e34e1ac/src/pyglimer/rf/moveout.py#L253)

These functions compute the differential traveltime for the relevant conversion in 1D and 3D, respectively. They take in a geographical location and an elevation. The elevation correction is implemented by extending the velocity model to the elevation in question, tracing the ray downward from that elevation using ray shooting given the ray parameter.
